repo/packages/Z/zeppelin/1/config.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "properties": {
+        "zeppelin": {
+            "type": "object",
+            "properties": {
+                "framework-name": {
```

Comment:
The config.json is base64 encoded and stored in a label of the marathon app. And yes. This is strange.